server/graphql_schemas/interest/schema.py

```diff
@@ -20,27 +20,27 @@ class Meta:
 class JoinCategory(relay.ClientIDMutation):
     """Join a category"""
     class Input:
-        category_id = graphene.ID(required=True)
+        categories = graphene.List(graphene.ID)
 
-    joined_category = graphene.Field(InterestNode)
+    joined_category_list = graphene.List(InterestNode)
 
     @classmethod
     def mutate_and_get_payload(cls, root, info, **input):
-        category_id = input.get('category_id')
+        categoryIdList = [category for category in input.pop('categories')]
```

Comment:
Please use `snake_case` for all variable names and accept `global_ids` instead in line 31.